src/twitter.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module "twitter-v2" {
+    export interface Credentials {
+        consumer_key?: string;
+        consumer_secret?: string;
+        bearer_token?: string;
+        access_token_key?: string;
+        access_token_secret?: string;
+    }
+
+    export interface TwitterResponse<T extends any> {
+        data: T;
+    }
+
+    export default class Twitter {
+        constructor(credentials: Credentials);
```

Comment:
We should also define a type for the `Credentials` class  (see [src/Credentials.js](https://github.com/HunterLarco/twitter-v2/blob/master/src/Credentials.js)) which is also exposed on the `Twitter` class under the field `credentials` (see [this line in the source](https://github.com/HunterLarco/twitter-v2/blob/master/src/twitter.js#L29)).